Austin V hw2

AustinV9121 opened the pull request.

AustinV9121 committed:
AustVStatCalc

standard functionality upgrade, fixed table oversizing on multiple selections, new file explorer 'choose file' option

AustinV9121 committed:
Merge branch 'master' of https://github.com/DesignOfExperimentsAtUTulsa/PyQTStatsCalculator

AustinV9121 committed:
TempSave

close but not done

AustinV9121 committed:
AustinVHW2

03 is the augmented code. Not sure if that is what was desired but I had to start to get it done by tomorrow.

jeremydaily reviewed (approved):
Looks fine. I'd suggest transforming temperature data to positive values to do the log normal, then shift it back.

AustinV9121 committed:
HW2 with new log plot

Added built in log plot, shifted to min temp value. Does not throw error for vals less than 0.
